.gitignore

```diff
@@ -167,3 +164,8 @@ cython_debug/
 
 # VSCode
 .vscode/
+
+# Ragna
+ragna.toml
+ragna/_version.py
+ragna.txt
```

Comment:
Is this also generated by the tests? If so, can we maybe configure the test to a use a tmp directory? As is, this seems kinda arbitrary.